src/main/java/sg/ncl/MainController.java

```diff
@@ -481,7 +487,7 @@ public String verifyEmail(@NotNull @RequestParam(ID) final String id,
         restTemplate.setErrorHandler(new MyResponseErrorHandler());
 
         final String link = properties.getSioRegUrl() + "/users/" + id + "/emails/" + emailBase64;
-        log.info("Activation link: {}, verification key {}", link, key);
+        // log.info("Activation link: {}, verification key {}", link, key);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)